### PR TITLE
Addon - Snapclient - Add snapserver hostname/IP setting

### PR DESCRIPTION
**Pull request by natumbri**
There is currently no way to tell a snapclient the hostname or IP of the snapserver it should connect to. Usually this isn't a problem; sometimes it is.

There is a "Host ID" setting, for setting the _ID_ of host running the snapclient. This is [apparently confusing some people](https://forum.libreelec.tv/thread/11817-snapcast/?postID=126566#post126566), who are looking for a way to set the _hostname_ or _IP_ of the host running the snapserver.

### CODE REVIEW

**Comment by litinoveweedle** [comment]
Ok, WIP for 7months for adding one parameter? Same problem still exists, no snapserver configuration. Could you please accept this PR? Thank you.

**Comment by CvH** [comment]
@awiouy any idea ?

**Comment by awiouy** [comment]
I will try to wrap this up this week-end

File: packages/addons/service/snapclient/source/bin/snapclient.start [hunk 1]
```diff
@@ -26,12 +26,14 @@ case "$LIBREELEC_ARCH" in
     ;;
 esac
 
+[ -n "$sc_ss" ] && sc_SS="--h $sc_ss"
```
Comment (MilhouseVH) [inline packages/addons/service/snapclient/source/bin/snapclient.start:29]: This is wrong - the option is either `-h` or `--host`, but not `--h`.

https://github.com/badaix/snapcast/blob/v0.15.0/client/snapclient.1#L34
https://manpages.ubuntu.com/manpages/disco/man1/snapclient.1.html

**Comment by awiouy** [comment]
I will pick this up in #4378

**Pull request closed without merging**